caluma/caluma_core/types.py

```diff
@@ -66,44 +66,60 @@ class DjangoConnectionField(DjangoConnectionField):
     This can be removed, when (or better if)
     https://github.com/graphql-python/graphql-relay-py/issues/12
     is resolved.
-
-    TODO: properly implement max_limit, see
-    https://github.com/graphql-python/graphene-django/blob/b552dcac24364d3ef824f865ba419c74605942b2/graphene_django/fields.py#L133
     """
 
     @classmethod
     def resolve_connection(cls, connection, args, iterable, max_limit=None):
+        # Remove the offset parameter and convert it to an after cursor.
+        offset = args.pop("offset", None)
+        after = args.get("after")
+        if offset:  # pragma: no cover
```

Comment:
I'd really like to see a test for this - it's a bit too complex to just "no cover" it IMHO

Reply:
Done